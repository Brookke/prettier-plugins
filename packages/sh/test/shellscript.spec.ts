import prettier from 'prettier'
import ShPlugin from 'prettier-plugin-sh'

test('fatal parse error with meaningful message', () => {
  expect(() =>
    prettier.format(`echo )`, {
      filepath: 'broken.sh',
      // @ts-ignore
      parser: 'sh',
      plugins: [ShPlugin],
    }),
  ).toThrow('a command can only contain words and redirects')
})
