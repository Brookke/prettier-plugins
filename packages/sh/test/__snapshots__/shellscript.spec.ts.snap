// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fatal parse error 1`] = `
"a command can only contain words and redirects
[0m[31m[1m>[22m[39m[90m 1 | [39mecho )[0m
[0m [90m   | [39m     [31m[1m^[22m[39m[0m"
`;
